Sources/FusumaViewController.swift

```diff
@@ -312,7 +312,7 @@ public struct ImageMetadata {
         }
     }
 
-    public override func viewWillDisappear(_ animated: Bool) {
+    public override func viewDidDisappear(_ animated: Bool) {
         super.viewWillDisappear(animated)
```

Comment:
@JonMo not sure if this'll matter or not, but we should change the `super` call too